### DISCUSSION TITLE
starting with i3

### DISCUSSION THREAD

**Discussion by davidon-top** [i3/i3#4519] [Q&A] [answered]
so i want to switch to i3wm but so far it hasnt been working rly
when i installed it with: apt install i3, it wont start or theres back screen
i tried to start it with gdm3: i3 and i3 with debig logs,  (idk where the debug logs are)
and with startx ~/.xinitrc:  exec i3
also when i finaly get it working is there a guide or this is the best i can get [https://i3wm.org/docs/userguide.html](https://i3wm.org/docs/userguide.html)

**Comment by davidon-top** [comment, marked as answer]
i fixed the black screen issue

**Reply by ghost** [reply]
When fixing something, please make sure to always share your solution before marking your post answered/solved. This might help some one else having similar issues.

**Reply by davidon-top** [reply]
the think is idk waht i did to fix the black screen think i just did stuff in the config and somehow somethink fixed it
